### PR TITLE
[blocked]feat: add blog post about the Gatsby swag store

### PR DESCRIPTION
**Pull request by jlengstorf**
[**Preview blog post**](https://deploy-preview-6873--gatsbyjs.netlify.com/blog/2018-07-16-swag-store/)

closes gatsbyjs/store.gatsbyjs.org#55

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit 64f6c4023e21082fbcd251420ce63c17902d7a12

https://app.netlify.com/sites/using-postcss-sass/deploys/5b5fa38bfdd72a79bed0f7ce

**Comment by KyleAMathews** [comment]
Deploy preview for *image-processing* failed.

Built with commit 64f6c4023e21082fbcd251420ce63c17902d7a12

https://app.netlify.com/sites/image-processing/deploys/5b5fa38cfdd72a79bed0f7d2

**Comment by KyleAMathews** [comment]
Deploy preview for *using-contentful* failed.

Built with commit 64f6c4023e21082fbcd251420ce63c17902d7a12

https://app.netlify.com/sites/using-contentful/deploys/5b5fa38cfdd72a79bed0f7d8

File: docs/blog/2018-07-16-swag-store/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+## What’s next?
+
+We have a lot of ideas for the Gatsby store — from [new swag ideas](https://github.com/gatsbyjs/store.gatsbyjs.org/issues?q=is%3Aissue+is%3Aopen+label%3A%22Swag+Ideas%22+sort%3Aupdated-desc) to adding more rewards for contributors who go above and beyond — but we’d love to hear more from you! Please [tweet at us][twitter] or open issues on [the store repo][swag-source] with ideas.
+
+For now, that’s it! We’ll leave you with my our favorite genre of tweet: “dogs wearing Gatsby swag”.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Thanks <a href="https://twitter.com/gatsbyjs?ref_src=twsrc%5Etfw">@gatsbyjs</a> for the t-shirt! If there are any developers out there that haven’t tried out Gatsby yet here is my dog Birch silently judging you. 😉 Trust me, it’s awesome. <a href="https://t.co/THDSmhKdBb">pic.twitter.com/THDSmhKdBb</a></p>&mdash; Ryan Wiemer (@ryanwiemer) <a href="https://twitter.com/ryanwiemer/status/1022267081998266368?ref_src=twsrc%5Etfw">July 25, 2018</a></blockquote>
```
Comment (shannonbux) [inline docs/blog/2018-07-16-swag-store/index.md]: wondering if this and other quotes tweets will show the image in final post? It'd be cool to see this dog w/o having to click on a link!

File: docs/blog/2018-07-16-swag-store/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+For more details and instructions on how to claim your free swag, see [the “swag for contributors” section of the docs][swag].
+
+Here’s the short version:
+
+1.  Head to the [swag store][store]
+2.  Log in with your GitHub account at the top right of the page
+3.  Request a discount code with your name and email
+4.  Use that discount code at checkout to get one free item from the swag store!
+
+> **NOTE:** The way this works is by checking that your GitHub account has at least one merged PR in the [Gatsby organization][org]. However, **contributing code is _not_ the only way to contribute to open source!** Check out [the “how to contribute” docs][contribute] for more examples of contributions that qualify for free swag. Email us at <mailto:team@gatsbyjs.com> if you’re a non-code contributor and want your free swag.
```
Comment (shannonbux) [inline docs/blog/2018-07-16-swag-store/index.md]: love this!

File: docs/blog/2018-07-16-swag-store/index.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+<figure>
+  <img alt="Gatsby swag posts on Twitter." src="./images/gatsby-swag-twitter.jpg" />
+  <figcaption>
+    Tweets from <a href="https://twitter.com/DeaNHtiD99/status/1023204484183416832">Dean</a>, <a href="https://twitter.com/ARebelBelle/status/1020044426712735744">Elle</a>, <a href="https://twitter.com/chaseadamsio/status/1021896138503245824">Chase</a>, and <a href="https://twitter.com/kato_katherine/status/1021967765400211456">Katherine</a> showing off their new Gatsby swag.
+  </figcaption>
+</figure>
+
+## Free swag for all contributors 💪💜
+
+We’re most excited about the launch of the store because it means contributors are now able to get stickers, socks, and/or t-shirts for free — a little token of our appreciation for putting the time in to make the open source community even better. You’re all amazing, and we 💜 you for it.
```
Comment (shannonbux) [inline docs/blog/2018-07-16-swag-store/index.md]: I'd take out the word *most* unless you plan on listing the other reasons we're excited

File: docs/blog/2018-07-16-swag-store/index.md (hunk 1 above)
Comment (jlengstorf) [reply docs/blog/2018-07-16-swag-store/index.md]: Once the preview finishes building, it _should_ show the images. I checked the build locally and it was working, so we'll see if that stays true. 😅

File: docs/blog/2018-07-16-swag-store/index.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/gatsbyjs?ref_src=twsrc%5Etfw">@gatsbyjs</a> is killing it in open source. This is how you treat contributors. Take notes 📝😊 <a href="https://t.co/9ta3IvfiZF">pic.twitter.com/9ta3IvfiZF</a></p>&mdash; Payton Burdette (@beardedpayton) <a href="https://twitter.com/beardedpayton/status/1020339698332512256?ref_src=twsrc%5Etfw">July 20, 2018</a></blockquote>
+
+## What’s next?
+
+We have a lot of ideas for the Gatsby store — from [new swag ideas](https://github.com/gatsbyjs/store.gatsbyjs.org/issues?q=is%3Aissue+is%3Aopen+label%3A%22Swag+Ideas%22+sort%3Aupdated-desc) to adding more rewards for contributors who go above and beyond — but we’d love to hear more from you! Please [tweet at us][twitter] or open issues on [the store repo][swag-source] with ideas.
+
+For now, that’s it! We’ll leave you with my our favorite genre of tweet: “dogs wearing Gatsby swag”.
```
Comment (m-allanson) [inline docs/blog/2018-07-16-swag-store/index.md]: _my_ or _our_?

File: www/package.json [hunk 5]
```diff
@@ -65,7 +65,10 @@
     "typeface-spectral": "^0.0.54",
     "typography": "^0.16.6",
     "typography-breakpoint-constants": "^0.15.10",
-    "typography-plugin-code": "^0.16.11"
+    "typography-plugin-code": "^0.16.11",
+    "zipkin": "^0.14.0",
+    "zipkin-javascript-opentracing": "^1.6.0",
+    "zipkin-transport-http": "^0.14.0"
```
Comment (m-allanson) [inline www/package.json]: Did you mean to add zipkin dependencies here?

**Comment by m-allanson** [comment]
Looks great :tada: I fixed a typo and added a couple of small comments.

File: www/package.json (hunk 5 above)
Comment (jlengstorf) [reply www/package.json]: Yeah, they allow us to run tracing on the www site. We can drop them if you prefer, though.

Comment (m-allanson) [reply www/package.json]: Nah all good, let's leave them in 👍

**Comment by m-allanson** [comment]
This is waiting for swag supplies to be replenished before it gets merged.

**Pull request merged**